conf/modules_nanopore.config

```diff
@@ -230,7 +248,7 @@ if (!params.skip_snpeff) {
             ]
         }
 
-        withName: 'TABIX_TABIX' {
+        withName: 'NFCORE_VIRALRECON:NANOPORE:.*:.*:.*:TABIX_TABIX' {
```

Comment:
Sorry if I'm being slow, we use `TABIX_TABIX` twice, so the above withName block is specific to the one used in VCFLIB_VCFUNIQ, and this one is for everywhere else?  Why the path with `snpeff` in it then?

Reply:
Ah, yeah. I guess this is a stylistic choice. So the one above after `VCFUNIQ` is invoked directly from the main workflow so I used the fully qualified name for the process. However, the one here is invoked via a sub-workflow so I have used the globs above as fillers to by tidier - we could have easily added in the full path. 

But in general, as long as you are able to discriminate between the two calls then it should be fine.

Reply:
I have some uncomfortable feels about amount of logic that happens in config files rather than in the workflows themselves with these new DSL2 patterns but I'll keep learning 😉

Reply:
Yeah, it has started bothering me after having re-written this pipeline and rnaseq. Loads of room for error with the logic in the config. The modules themselves are beautifully tidy now but we have shifted the complexity somewhere else it seems. Definitely better for things to evolve this way around though in general.